src/solvers/flattening/boolbv_get.cpp

```diff
@@ -262,14 +268,14 @@ exprt boolbvt::bv_get_rec(
   }
   }
 
-  return nil_exprt();
+  UNREACHABLE;
```

Comment:
There should be a PRECONDITION in the if(...) above on the type, or a failure here will be hard to debug.

Reply:
I have borrowed a bit from #2548.